client/core/core.go

```diff
@@ -6281,6 +6320,14 @@ func (c *Core) tipChange(assetID uint32, nodeErr error) {
 // updated. If there is no synced book, but a non-zero fee suggestion is already
 // cached, no new requests will be made.
 func (c *Core) cacheRedemptionFeeSuggestion(t *trackedTrade) {
+	if rater, is := t.wallets.toWallet.feeRater(); is {
+		feeRate, err := rater.FeeRate()
+		if err == nil {
+			atomic.StoreUint64(&t.redeemFeeSuggestion, feeRate)
+			return
+		}
+		c.log.Debugf("unable to retrieve fee rate from FeeRater. falling back to other methods: %v", err)
```

Comment:
Hmm, with DCR in SPV mode, we constantly spam the logs, (I have a buy order booked, if that matters):

```
2022-02-02 13:58:34.660 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:42.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:45.003 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588795
2022-02-02 13:58:49.660 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:57.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:00.003 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588796
2022-02-02 13:59:04.660 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:12.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:15.003 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588797
2022-02-02 13:59:19.659 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:27.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:30.002 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588798
2022-02-02 13:59:34.660 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:42.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:45.002 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588799
2022-02-02 13:59:49.659 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:59:57.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 14:00:00.003 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588800
2022-02-02 14:00:04.659 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 14:00:12.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 14:00:15.004 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588801
2022-02-02 14:00:19.659 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 14:00:27.159 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 14:00:30.004 [TRC] CORE: notify: |DATA| (epoch) - Index: 109588802
2022-02-02 14:00:34.659 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 14:00:42.160 [DBG] CORE: unable to retrieve fee rate from FeeRater. falling back to other methods: -9: RPC passthrough requires dcrd RPC synchronization
```

Reply:
This is reminiscent of the `Rescanner` interface issue with BTC where it was necessary to split `ExchangeWallet -> {`baseWallet`,`ExchangeWalletSPV`,`ExchangeWalletFullNode`}.
We might have to do the same for DCR, but hopefully we can just ignore an error from `FeeRate`, at least in `cacheRedemptionFeeSuggestion` .

Reply:
Bump on this DCR SPV issue.  This and the non-functional `dc.fetchFeeRate` for `PayFee` are the only real show stoppers with this PR.